package.json

```
@@ -19,7 +20,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --coverage",
```

Please add new command for coverage and don't disturb existing command.

```suggestion
    "test": "react-scripts test --coverage",
```
```suggestion
    "test": "react-scripts test --coverage",
```
```suggestion
    "test:coverage": "react-scripts test --coverage",
```